root/client/src/components/poll/adminPage.js

```diff
@@ -0,0 +1,134 @@
+        </h1>
+        <h2>{name}</h2>
+        <div id='header-desktop-poll-btns'>
+          <ul id="ul-desktop-poll-btns">
+            <li>
+              <button id='btn-admin-mobile-edit' onClick={(e) => handleUpdate(e)}>
+                Edit Poll
+              </button>
+            </li>
+            <li>
+              <button id='btn-admin-mobile-delete' onClick={handleDelete}>
+                Delete Poll
+              </button>
+            </li>
+          </ul>
+        </div>
+        <div id="header-poll-btns">
+          <button id='btn-mobile-open' onClick={handleOpenDescription} ref={btnMobileOpen}>
+            <svg xmlns="http://www.w3.org/2000/svg" height="16" width="14" viewBox="0 0 448 512"><path d="M256 80c0-17.7-14.3-32-32-32s-32 14.3-32 32V224H48c-17.7 0-32 14.3-32 32s14.3 32 32 32H192V432c0 17.7 14.3 32 32 32s32-14.3 32-32V288H400c17.7 0 32-14.3 32-32s-14.3-32-32-32H256V80z" fill="#FFFFFF"/></svg>
```

Comment:
have you considered using a font library instead of manually hard-coding all of these svgs?

Reply:
I have not considered using a font library, i find hard-coding SVG's not a major issue. Is there an issue with it, something to worry about in your opinion?

Reply:
for a small scale app, no, nothing to worry about. but for larger apps where usability and readability comes to mind, then using a library is often preferred. this is _generally_ the case in the workplace too, so I only suggested it to give you more familiarity of it in the future.

Reply:
For this project, I'll keep with the hard-code. For my next project, i'll make sure to use a font library. Thanks Dan.